ext/odbc/php_odbc.c

```diff
@@ -2195,42 +2235,43 @@ void odbc_do_connect(INTERNAL_FUNCTION_PARAMETERS, int persistent)
 		persistent = 0;
```

Comment:
Should this maybe warn in case the persistent connection version is used?

Reply:
Hm, yes, it could warn when persistent connections are not allowed. I can deal with it in a followup